src/index.mjs

```diff
@@ -48,13 +37,11 @@ function getInstallCommand(pkgManager) {
  */
 function createConfig(cwd, pkgManager) {
 	const spinner = ora('Creating config...').start();
-	const existingConfigs = getExistingConfigsInDirectory(cwd);
+	const existingConfigs = getExistingConfigsInDirectory();
 
-	if (existingConfigs.length > 0) {
+	if (existingConfigs !== null) {
 		spinner.fail(
-			`Failed to create config:\nThe ${existingConfigs.join(
-				', ',
-			)} config(s) already exist. Remove them and then try again.`,
+			`Failed to create config:\nThe ${existingConfigs.filepath} config(s) already exist. Remove them and then try again.`,
```

Comment:
[suggest] How about making the error message more understandable like this?

- Show a file base name instead of a full path (too long)
- Remove no longer needed `(s)`
- Use double quotes for a file name

```suggestion
			`Failed to create config:\nThe "${path.basename(existingConfigs.filepath)}" config already exists. Remove them and then try again.`,
```

Before:

```
✖ Failed to create config:
The /Users/masafumi/git/stylelint/create-stylelint/.stylelintrc.json config(s) already exist. Remove them and then try again.
```

After:

```
✖ Failed to create config:
The ".stylelintrc.json" config already exists. Remove them and then try again.
```

Reply:
In addition, when `package.json` includes the `stylelint` property, the error message is the following:

```
✖ Failed to create config:
The "package.json" config already exists. Remove them and then try again.
```

The message may be misleading. For example, can you improve the message like this?

```
✖ Failed to create config:
The "stylelint" config in "package.json" already exists. Remove them and then try again.
```